composables/zksync/useWithdrawalFinalization.ts

```diff
@@ -55,28 +46,31 @@ export default (transactionInfo: ComputedRef<TransactionInfo>) => {
       "0x7726827caac94a7f9e1b160f7ea819f172f7b6f9d2a97f992c38edeab82d4110",
       provider
     );
-    const { l1BatchNumber, l2MessageIndex, l2TxNumberInBlock, message, proof } = await wallet.finalizeWithdrawalParams(
-      transactionInfo.value.transactionHash
-    );
-    return {
-      chainId: await retrieveChainId(),
-      l1BatchNumber,
-      l2MessageIndex,
-      l2TxNumberInBlock,
-      message,
-      proof,
-    };
+    return await wallet.getFinalizeWithdrawalParams(transactionInfo.value.transactionHash);
   };
 
   const getTransactionParams = async () => {
     finalizeWithdrawalParams.value = await getFinalizationParams();
+    const provider = providerStore.requestProvider();
+    const chainId = BigInt(await provider.getNetwork().then((n) => n.chainId));
```

Comment:
Shouldn't we use chainId from the config? This way it's more obvious if it's incorrect + one less request.